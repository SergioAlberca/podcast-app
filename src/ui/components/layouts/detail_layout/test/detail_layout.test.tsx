import { renderWithProviders } from "@/test/test_utils";

import { BrowserRouter } from "react-router-dom";
import DetailLayout from "../detail_layout";

describe("MainLayout component tests", () => {
  it("render component", () => {
    const { container } = renderWithProviders(
      <BrowserRouter>
        <DetailLayout podcastId="1234">
          <p>Fake children</p>
        </DetailLayout>
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
