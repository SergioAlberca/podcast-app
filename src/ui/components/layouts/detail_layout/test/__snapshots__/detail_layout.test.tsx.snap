// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MainLayout component tests > render component 1`] = `
<div>
  <div
    class="_podcast-detail-container_d37adf"
  >
    <p>
      Fake children
    </p>
  </div>
</div>
`;
